cmd/provider_cmd_squadcast.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"log"
+
+	squadcast_terraforming "github.com/GoogleCloudPlatform/terraformer/providers/squadcast"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraformutils"
+	"github.com/spf13/cobra"
+)
+
+func newCmdSquadcastImporter(options ImportOptions) *cobra.Command {
+	var refreshToken string
+	var teamName string
+
+	cmd := &cobra.Command{
+		Use:   "squadcast",
+		Short: "Import current state to Terraform configuration from Squadcast",
+		Long:  "Import current state to Terraform configuration from Squadcast",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			originalPathPattern := options.PathPattern
+			for _, region := range options.Regions {
```

Comment:
we don't need to loop over the region. refer to other providers for reference(`vault`)
Though we have multiple dc, app instances are separately hosted.

Reply:
Initially we used this approach because the argument of `newCmdSquadcastImporter` had an options argument which had a Region parameter ([]string{}), but now we switched it to a variable (string)